test/IntegrationTests/AspNetTests.cs

```diff
@@ -122,12 +122,11 @@ public async Task SubmitMetrics()
     private async Task<IContainer> StartContainerAsync(int webPort, string? appPoolMode = null)
     {
         // get path to test application that the profiler will attach to
-        string imageName = appPoolMode == "Classic" ? "testapplication-aspnet-netframework-classic" : "testapplication-aspnet-netframework";
+        var imageName = appPoolMode == "Classic" ? "testapplication-aspnet-netframework-classic" : "testapplication-aspnet-netframework";
 
-        string networkName = DockerNetworkHelper.IntegrationTestsNetworkName;
-        string networkId = await DockerNetworkHelper.SetupIntegrationTestsNetworkAsync();
+        _ = await DockerNetworkHelper.SetupIntegrationTestsNetworkAsync();
```

Comment:
Can we refactor the code so that `DockerNetworkHelper.SetupIntegrationTestsNetworkAsync` returns `DockerNetworkHelper.IntegrationTestsNetworkName`?